src/factory.js

```diff
@@ -124,21 +133,25 @@ export default function factory(React, empty) {
           } else if(isAuthenticating) {
             return <LoadingComponent {...otherProps} />
           } else {
-            // Don't need to display anything because the user will be redirected
-            return React.createElement(empty)
+            // Unauthorized: Display FailureComponent or nothing
+            // if FailureComponent is undefined the user will never see
+            // this because they will be redirected to failureRedirectPath
+            return FailureComponent ? <FailureComponent authData={authData} {...otherProps} /> : React.createElement(empty)
```

Comment:
one catch here, I think empty isn't actually set to be null. Its a [div](https://github.com/mjrussell/redux-auth-wrapper/blob/master/src/index.js#L4) 😕  Seems like a `<div />` might trip people up if they expect it to be hidden. Should we change the default empty?

I believe I used a div initially because React Router can get angry if you use null. I can verify quickly though